source/flow.rst

```diff
@@ -599,6 +598,67 @@ Fields:
 * **Name** *(optional)*: Name of the node
 
 
+Publish in FTP topic
+********************
+
+.. _kafka2ftp_node:
+.. image:: images/nodes/kafka2ftp_node.png
+    :width: 20%
+    :align: left
+    :alt: kafka2ftp_node
+
+Node to forwarding messages to Apache Kafka FTP topic.
+
+It publish to the topic ``tenant.dojot.ftp`` (*tenant* is defined by which tenant the flow belongs to)
+where in which messages are produced with information about the file name,
+encoding format and file content.
+
+.. _kafka2ftp_node_cfg:
+.. figure:: images/nodes/kafka2ftp_node_cfg.png
+    :width: 50%
+    :align: center
+    :alt: kafka2ftp_node_cfg
+
+    : *Publish in FTP topic* configuration
+
+Fields:
+
+* **Encoding** *(required)*: Encoding the contents of the file to be sent. Valid values are: ascii, base64, hex, utf16le, utf8 and binary.
+* **Filename** *(required)*: Variable with the name of the file to be sent.
+* **Content** *(required)*: Variable with the file contents to be sent.
+* **Name** *(optional)*: Name of the node
+
+
+Example of message sent by this node below:
```

Comment:
Example of message sent by this node below: -> Example of a message sent by this node: